terraform/modules/cloudwatch_s3_ingestor/eventbridge.tf

```
@@ -0,0 +1,28 @@
+resource "aws_cloudwatch_event_rule" "rds_log_group_creation_broker" {
+  for_each    = toset(var.environments)
+  name        = "rds-log-group-creation-broker"
+  description = "Triggers when a CloudWatch Log Group is created with an /aws/rds/instance/cg-aws-broker-dev prefix in the name."
+  event_pattern = jsonencode({
+    "source"      = ["aws.logs"],
+    "detail-type" = ["AWS API Call via CloudTrail"],
+    "detail" = {
+      "eventSource" = ["logs.amazonaws.com"],
+      "eventName"   = ["CreateLogGroup"],
+      "requestParameters" = {
+        "logGroupName" = [
+          {
+            prefix = "/aws/rds/instance/cg-aws-broker-${each.key == "production" ? "prd" : (each.key == "staging" ? "stg" : "dev")}"
```

```suggestion
            prefix = "/aws/rds/instance/cg-aws-broker-${locals.prefixes[each.key]}"
```

And add this in to the file:

Let's do this

```
locals {
  prefixes = {
    "production": "prd",
     "staging": "stg",
     "development": "dev"
  }
}
```

---

interesting, i dig that